entries/g/gumroad.com.json

```diff
@@ -0,0 +1,14 @@
+{
+    "Gumroad": {
+      "domain": "gumroad.com",
+      "tfa": [
+        "email"
+      ],
+      "documentation": "https://help.gumroad.com/article/292-account-login-security",
+      "notes": "2FA is on by default for all users",
```

Comment:
This information is not of much use for users as the documentation page already explains this.
```suggestion
```